net-dialup/xl2tpd/xl2tpd-1.3.12.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit systemd toolchain-funcs
+
+DESCRIPTION="A modern version of the Layer 2 Tunneling Protocol (L2TP) daemon"
+HOMEPAGE="http://www.xelerance.com/services/software/xl2tpd/"
```

Comment:
Please include the verbatim GitHub homepage here, so we could easily copy-paste it into addressbar ;-).

Reply:
Good point, the formerly linked page is outdated anyway.